coding/cpp_style_guide.rst

```diff
@@ -1984,432 +1982,64 @@ We should, of course, strive to minimize the amount of such information.
 
 .. _style-guide-cpp-6-1:
 
-6-1. Multiple statements per line SHOULD NOT be used.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. code-block:: cpp
+6-1. We follow the clang-format Google style for code layout with a few exceptions.
```

Comment:
Aren't we saying that we follow the Google style for code layout with a few exceptions (in which case a link to the Google style guide would be very useful)? clang-format is a tool that can be used to ensure that the style is followed. Or are we actually saying that clang-format with the supplied configuration file is the definition of our standard rather than a tool that can be used to enforce it?